tests/kafkatest/tests/security_rolling_upgrade_test.py

```diff
@@ -53,37 +50,38 @@ def create_producer_and_consumer(self):
 
         self.consumer = ConsoleConsumer(
             self.test_context, self.num_consumers, self.kafka, self.topic,
-            consumer_timeout_ms=60000, message_validator=is_int, new_consumer=True)
+            consumer_timeout_ms=60000, message_validator=is_int, new_consumer=True, log_values=True)
 
         self.consumer.group_id = "unique-test-group-" + str(random.random())
+        self.consumer.log_level = "TRACE"
```

Comment:
@granders This was deliberate, but on consideration I've made this ubiquitous. Trace logging in the consumer is not that verbose. What is verbose is the logging of messages from the console consumer (now as the default). This is pretty useful when debugging problems with the consumer which we've had a few of recently. I'm running a full build now. We can see how bad the damage is :)